docs/ops/movement/Gather_7.md

```diff
@@ -0,0 +1,167 @@
+## Gather <a name="Gather"></a> {#openvino_docs_ops_movement_Gather_7}
+
+**Versioned name**: *Gather-7*
+
+**Category**: Data movement operations
+
+**Short description**: *Gather* operation takes slices of data in the first input tensor according to the indices
```

Comment:
`of data in the first` -> `of data of the first`

Reply:
done.